pkg/datapath/linux/config/config.go

```diff
@@ -532,6 +534,32 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 		cDefinesMap["ENABLE_CUSTOM_CALLS"] = "1"
 	}
 
+	if option.Config.EnableVTEP {
+		cDefinesMap["ENABLE_VTEP"] = "1"
+
+		l := len(option.Config.VtepEndpoints)
+		cDefinesMap["VTEP_NUMS"] = strconv.Itoa(l)
+
+		ip_str := "(__u32[]){"
+		mac_str := "(__u64[]){"
+
+		for i, ep := range option.Config.VtepEndpoints {
+
+			ip_str = ip_str + fmt.Sprintf("%#x", byteorder.NetIPv4ToHost32(ep)) + ", "
+
+			mac := option.Config.VtepMACs[i]
+			//append two bytes to avoid binary LittleEndian error complain
+			zero := []byte{0x00, 0x00}
+			vtep_mac := append(mac, zero...)
+			num := binary.LittleEndian.Uint64(vtep_mac)
+			mac_str = mac_str + fmt.Sprintf("%#x", num) + ", "
```

Comment:
silly me :-), I did not realize `MAC.Uint64()` does the same translation, above is ugly hack. following fixed it

```
@@ -548,11 +547,11 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
                        ip_str = ip_str + fmt.Sprintf("%#x", byteorder.NetIPv4ToHost32(ep)) + ", "
 
                        mac := option.Config.VtepMACs[i]
-                       //append two bytes to avoid binary LittleEndian error complain
-                       zero := []byte{0x00, 0x00}
-                       vtep_mac := append(mac, zero...)
-                       num := binary.LittleEndian.Uint64(vtep_mac)
-                       mac_str = mac_str + fmt.Sprintf("%#x", num) + ", "
+                       vtep_mac, err := mac.Uint64()
+                       if err != nil {
+                               return err
+                       }
+                       mac_str = mac_str + fmt.Sprintf("%#x", vtep_mac) + ", "
 

```